server/cli/src/cli.rs

```diff
@@ -396,6 +398,77 @@ async fn main() -> anyhow::Result<()> {
             info!("Refresh data result: {:#?}", result);
         }
         Action::SignPlugin { path, key, cert } => sign_plugin(&path, &key, &cert)?,
+        Action::UpsertStandardReports => {
+            let connection_manager = get_storage_connection_manager(&settings.database);
```

Comment:
This cli.rs file is getting huge! Would be nice to break it up a bit. Even if we just do it for this new report for now. 
Perhaps we do a re-arrange PR separately to avoid a huge PR review :)